common/pipeline_storage/src/test/scala/weco/pipeline_storage/ImageIndexableTest.scala

```diff
@@ -107,7 +106,7 @@ class ImageIndexableTest
   private def withImagesIndexAndIndexer[R](
     testWith: TestWith[(Index, ElasticIndexer[Image[ImageState.Augmented]]), R]
   ) =
-    withLocalAugmentedImageIndex { index =>
+    withLocalAugmentedImageIndex { index:Index =>
```

Comment:
```suggestion
    withLocalAugmentedImageIndex { index: Index =>
```

why hasn't the autoformatter picked this up?

(there were a few other spots)